boring/src/ssl/mod.rs

```diff
@@ -1707,6 +1732,33 @@ impl SslContextBuilder {
         }
     }
 
+    fn set_safe_defaults(&mut self) -> Result<(), ErrorStack> {
+        // Choose a curves list suitable for the given feature set.
```

Comment:
@bwesterb can you provide some guidance here? What do you think would be make sensible defaults?